### PR TITLE
Adding doc about running tests with code coverage

### PR DESCRIPTION
**Pull request by ballinette**

### CODE REVIEW

**Comment by scottinet** [comment]
+1

File: features/README.md [hunk 1]
```diff
@@ -10,4 +10,10 @@ See the [Getting Started](../README.md) documentation for more information about
 
     $ npm run functional-testing
 
-If your Kuzzle instance use a different configuration than the default one, you can make these tests point at other locations by editing the `.kuzzlerc` file at the root directory.
\ No newline at end of file
+If your Kuzzle instance use a different configuration than the default one, you can make these tests point at other locations by editing the `.kuzzlerc` file at the root directory.
```
Comment (benoitvidis) [inline features/README.md]: If your Kuzzle instance use**s** a different configuration

File: features/README.md [hunk 2]
```diff
@@ -10,4 +10,10 @@ See the [Getting Started](../README.md) documentation for more information about
...
+
+## Running Functional Tests and report test coverage
+
+    $ npm run functional-testing && npm run crawl-coverage
+
+Coverage report will be stored in HTML human readable format : ```coverage/features/coverage.html```
```
Comment (benoitvidis) [inline features/README.md]: No space before colons

File: features/README.md (hunk 1 above)
Comment (ballinette) [inline features/README.md]: fixed

File: features/README.md (hunk 2 above)
Comment (ballinette) [inline features/README.md]: fixed

File: test/README.md [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
 
     $ npm run unit-testing
-    
+
+## Running Unit Tests and report test coverage
+
+    $ npm run unit-testing --coverage
+
+Coverage report will be stored in ```coverage/``` directory:
+* in json format : ```coverage/coverage.json```
```
Comment (benoitvidis) [inline test/README.md]: sorry, still some colons here :\

Comment (ballinette) [inline test/README.md]: you lie ;)

**Comment by benoitvidis** [comment]
+1

**Pull request merged**